docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -729,6 +729,28 @@ p2p-host="0.0.0.0"
 Specifies the host on which P2P listens.
 The default is 127.0.0.1.
 
+### p2p-interface
+
+```bash tab="Syntax"
+--p2p-interface=<HOST>
+```
+
+```bash tab="Command Line"
+--p2p-interface=192.168.1.132
+```
+
+```bash tab="Environment Variable"
+BESU_P2P_INTERFACE=192.168.1.132
+```
+
+```bash tab="Configuration File"
+p2p-interface="192.168.1.132"
+```
+
+Specifies the network interface on which the node listens for [P2P communication](../../HowTo/Find-and-Connect/Configuring-Ports.md#p2p-networking).
+Use the option to specify the required network interface when the device that Besu is running on 
+has multiple network interfaces. The default is 0.0.0.0.
```

Comment:
Same comment as above.  I think referencing the default `0.0.0.0` is okay as long as we indicate that it means all interfaces:
```suggestion
has multiple network interfaces. The default is 0.0.0.0 (all interfaces).
```

Reply:
Fixed.